src/hiero_sdk_python/consensus/topic_update_transaction.py

```
@@ -120,7 +123,9 @@ def set_submit_key(self, key: PublicKey) -> "TopicUpdateTransaction":
         self.submit_key = key
         return self
 
-    def set_auto_renew_period(self, seconds: Union[Duration, int]) -> "TopicUpdateTransaction":
+    def set_auto_renew_period(
+        self, seconds: Union[Duration, int]
```

Ruff (0.14.14)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)